<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>screenlogin</class>
 <widget class="QDialog" name="screenlogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TicTacToe</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="UserName1">
      <property name="text">
       <string>UserName1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="LineEditUserName1"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="Password1">
      <property name="text">
       <string>Password1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="LineEditPassword1"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="loginpushButton">
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="showscreenbuttonlogin">
      <property name="text">
       <string>ShowScore and Login</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="signuppushButton">
      <property name="text">
       <string>Signup</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="screenloginMainwindowbutton">
      <property name="text">
       <string>Back to MainMenu</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="quitpushButton">
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="UserName2">
      <property name="text">
       <string>User Name2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="LineEditUserName2"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="Password2">
      <property name="text">
       <string>Password2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="LineEditPassword2"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
